SPECS/qemu/qemu.spec

```diff
@@ -2294,6 +2298,9 @@ useradd -r -u 107 -g qemu -G kvm -d / -s %{_sbindir}/nologin \
 
 
 %changelog
+* Tue Dec 20 2022 Nan Liu <liunan@microsoft.com> - 6.2.0-13
+- Adress CVE-2021-3929, CVE-2021-4207
```

Comment:
Address* (spelling)

Reply:
Ah-Ha, I didn't notice that. Thanks for pointing it out. Fixed all the spelling issues.